weather_stats/weatherstatsapi/weather/openweather/schemas.py

```diff
@@ -0,0 +1,60 @@
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+
+class GetWeatherParams(BaseModel):
+    lat: float
+    lon: float
+    units: str = "metric"
+
+
+class GetWeatherData(BaseModel):
+    class _Clouds(BaseModel):
```

Comment:
As its harder to work with typings on nested classes, consider moving them at root level, hence renaming to `RawWeather*` would be better.

Reply:
What problems do you have in mind? I don't want anyone to use those nested Models explicitly.

Reply:
As discussed offline, this would change API, which is not desired.